docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
+- **Aggregate Function** - Selects the type of aggregate to perform
+- **Sort Field** - Sets the collection field you wish to order results by
+- **Filter** - Sets filters to only only run metrics on items that meet some criteria
+
+::: warning Must be numeric Non-numeric fields like text cannot be aggregated, so they become grayed out and
+unselectable in the field dropdown list. :::
+
+#### Style and Format Options
+
+- **Abbreviate Value** - Abbreviates large numbers with a letter _(e.g. 2,000 = 2k)_
+- **Decimals** - Changes the number of decimals displayed
+- **Prefix** - Adds text before the metric
+- **Suffix** - Adds text after the metric
+- **Conditional Styles** - Changes color of number displayed if it is `>`, `>=`, `<`, `<=`, `==` (equal to), or `!=`
+  (not equal to) to some defined value
+
+### Time Series Overview
+
+The Time Series panel allows you to build graphs and see how data changes over time.
+
+::: tip Note In order to use this panel, your collection will need a datetime field. :::
+
+![image of Time Series panel options](image.jpg)
+
+- **Collection** - Selects the collection you want to analyze
+- **Color** - Sets color of your graph curve
+- **Group Aggregation** - Selects the type of aggregation to perform
+- **Group Precision** - Sets the time interval to aggregate by _(days, weeks, months etc.)_
+- **Date Field** - Selects the datetime field used along the x-axis.
+- **Date Range** - Selects the time range from which data should be displayed
+- **Value Field** - Sets the field you wish to aggregate
+- **Value Decimals** - Changes the number of decimals displayed on the y-axis
+- **Min Value** - Sets minimum value displayed on y-axis
+- **Max Value** - Sets maximum value displayed on y-axis
+- **Curve Type** - Sets curve to be shown as smooth, straight or stepline
+- **Fill Type** - Sets fill type under curve to gradient, solid, or none
+- **Filter** - Allows filtering of items considered for analysis based on logical conditions
+- **Show X-axis** - Toggles display of time along X-axis
+- **Show Y-axis** - Toggles display of numeric values along Y-axis
+
+### Advanced Filtering
+
+The filtering options are infinitely customizable. However, they will be limited by the collections, relations, and
+information stored in your database.
+
+#### Nested Filters
+
+Sometimes, you will need to run filters that require information from two or more collections. Here might be a few
+examples:
+
+- Highest paid authors that wrote 5 or more books (uses `authors` and `books` collections)
+- Products with less than 50 sales monthly (uses `products` and `orders` collections)
+- Blog Posts with the highest number of social shares (uses `blog` and `social share metadata` collections )
+
+Nested filters limit results of one collection by the values of fields from a related collection. Let's look at how to
+create a nested filter:
+
+![Image of a nested filter with numbers](image.jpg)
+
+1. Create a filter
```

Comment:
Again, Filters are a pretty global thing, we might want to have a really solid explanation somewhere, and link there. We can still give contextual guidance on this page, but should link to "learn more" within the main section

Reply:
I expanded the filters section earlier this evening. But honestly, I started to feel the scope started ballooning like crazy and cut it with some (expanded but) basic explanations.

I'm assuming the readers have zero SQL knowledge, but decent math, actively do analytics, and are pretty fire in excel.... Is that about right for these dashboard creators?

I'd like to take the approach that _the readers know what filters are and just need a primer on how ours work_ in the main docs... Then get super-duper granular in a 1 to 4 hour tutorial series... Let me know your thoughts on that strategy!

Reply:
Anything within the Docs "App Guide" should be geared towards anyone (non-technical too) as a baseline, but with hints and details that help out the more technical power-users and admins.

> I'd like to take the approach that the readers know what filters are and just need a primer on how ours work in the main docs... Then get super-duper granular in a 1 to 4 hour tutorial series... Let me know your thoughts on that strategy!

Pretty close! I like this, and we can include permission tutorials for a bunch of subjective use-case examples. However, I do want to have dedicated sections in the Docs that thoroughly explain working with both permissions and filters (separately).

In general, we should have ONE dedicated and comprehensive place in the Docs for each feature/component/concept. Then, we reference/link to those authoritative sections throughout the docs as needed. For example, here in Insights (context, App Guide) we would simply mention Filters (maybe with 1+ relevant examples for this context)... but would link to the main section on how to work with Filters (authoritative, Configuration).

Make sense? This is chicken and the egg, since we'll need to make sure we have all those single-source-of-truth docs written first, so that we can link to them.

Reply:
Yeah, I get it! I mentioned to Rijk and Nils that I would love to add some dictionary table(s) DB that holds all of our terms and lets a user hover over some term and get a tooltip definition and "learn more" button. That's obviously a ways out. But I digress- "long story long", I think this approach is great. However, it will mean multiple and constant rounds of docs updates as more and more pages get added over the next several months (which I am totally happy about).